python/src/trezorlib/cli/solana.py

```diff
@@ -65,6 +65,12 @@ def sign_tx(
     additional_information = None
     if additional_information_file:
         raw_additional_information = json.load(additional_information_file)
+
+        definitions = raw_additional_information.get("definitions", None)
+        if definitions:
+            token = definitions.get("encoded_token", None)
+            definitions = messages.SolanaDefinitions(encoded_token=token)
```

Comment:
```suggestion
            if token:
                definitions = messages.SolanaDefinitions(
                    encoded_token=bytes.fromhex(token)
                )
```

This way a token definition is e.g. `74727a643102ffffffff64000a20e303bf2ce35eddf29e99210c22a5203541ea2e3313bfed187a500eaf1b7e3560122b546f6b656e6b65675166655a79694e77414a624e62474b50465843577542766639537336323356513544411a0a46616b6520546f6b656e220746616b65546f6b000746efd16f0dbb3b452ba95ff78dd11596fe49d85ab27fbf18e35a61da5f9c76708e8f58302c4e6c27d58c6a9cc440ae2c28b710b1e095d45ac3f490013da8bc04` - no escaped characters and it's the same format as transaction when signing via cli.

To print it from the tests `print(common.make_solana_defs(token).encoded_token.hex())`.